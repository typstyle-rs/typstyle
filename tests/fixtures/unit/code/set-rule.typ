
#set text(red)

#set circle[666]
#circle()
#circle[]
#set circle()
#circle()
#set circle(stroke: blue)[777]
#circle()
