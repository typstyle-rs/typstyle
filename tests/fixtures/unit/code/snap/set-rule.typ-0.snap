---
source: tests/src/unit.rs
input_file: tests/fixtures/unit/code/set-rule.typ
---
// DUMMY

#set text(
  red,
)

#set circle[666]
#circle()
#circle[]
#set circle()
#circle()
#set circle(
  stroke: blue,
)[777]
#circle()
